### DISCUSSION TITLE
Define table for RoutingModel in Model

### DISCUSSION THREAD

**Discussion by freshface** [statamic-rad-pack/runway#490] [Feature Requests]
When using use RunwayRoutes; is there a way to set the used table the RoutingModel uses in my own model?

**Comment by duncanmcclean** [comment]
Just to clarify... are you wanting to customise the name of the `runway_uris` table?

**Reply by duncanmcclean** [reply]
If that *is* what you're wanting to customize, it's now possible: https://runway.duncanmcclean.com/frontend-routing#content-customizing-the-table-name
